src/3rd_party-static/CMakeLists.GTest

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.2)
```

Comment:
@mkorniichuk may be 2.8.11?

Reply:
@mkorniichuk  please add Copyright

Reply:
@mkorniichuk I'm not sure we need this specifier in this file.

Reply:
@BSolonenko this cmake list will be triggered as a standalone project, so the **cmake_minimum_required** is a required statement. BTW, in such a way, we specify the minimum cmake version supported by gtest.